larky/src/test/resources/quick_tests/test_justin.star

```diff
@@ -41,17 +41,17 @@ def _sign_header_and_claims(encoded_header, encoded_claims, algorithm, key):
     k = jwk.construct(key, algorithm)
     encoded_signature = base64url_encode(k.sign(signing_input))
     encoded_string = b".".join([encoded_header, encoded_claims, encoded_signature])
-    print(encoded_string)
+    # print(encoded_string)
     return encoded_string
 
 
 def sign(payload, key, headers=None, algorithm=ALGORITHMS.HS256):
     encoded_header = _encode_header(algorithm, additional_headers=headers)
     encoded_payload = _encode_payload(payload)
-    print("======================")
-    print(encoded_header)
-    print(encoded_payload)
-    print("======================")
+    # print("======================")
```

Comment:
Are those debug leftovers?

Reply:
Yeah, I just wanted to stop the print output during test execution. I didn't write this `test_justin.star` but I don't want test output polluted by `print()` statements.